plugins/workspace-backpack/src/ui_events.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview UI events used for Backpack.
+ * @author kozbial@google.com (Monica Kozbial)
+ */
+
+import * as Blockly from 'blockly/core';
+
+/**
+ * Name of event that records a backpack open.
+ * @const
+ */
+export const BACKPACK_OPEN = 'backpack_open';
```

Comment:
Do these have to be exported separately? I'm curious why you made them separate instead of a property on the event class or something. It doesn't look like you use this outside of this file so I'm not sure why you need to export it like this, but I'm not 100% sure on the requirements of registering the event

Reply:
For other events in Blockly we also export the type constant separately 
It's needed to make checks like:
```
if (event.type === BACKPACK_OPEN) {
}
```
This property (`type`) is defined on the prototype, so it wouldn't be "right" to do something like:
```
if (event.type === BackpackOpen.protoype.type) {
}
```

Reply:
I see, thanks for the explanation!